src/components/modals.ts

```diff
@@ -61,8 +62,24 @@ abstract class OmnisearchModal extends Modal {
 
     // #endregion Up/Down navigation
 
+    let openInCurrentPaneKey,
```

Comment:
- Please don't use multiple declaration, use 1 let per variable declaration. Just a personal preference, and to stay in line with the repo's style
- You need to type the variables, e.g `let openInCurrentPaneKey: Modifier[]`

Reply:
Done